src/Microsoft.HttpRepl.Tests/Commands/CommandTestsBase.cs

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.Net.Http;
 using Microsoft.HttpRepl.Fakes;
 using Microsoft.HttpRepl.FileSystem;
```

Comment:
This using appears to not be necessary anymore.

Reply:
Its coming from IDictionary in ln 28

Reply:
I'm referring to `using Microsoft.HttpRepl.FileSystem;`

Reply:
Ah sorry. I removed it in one of the previous commits:
https://github.com/aspnet/HttpRepl/commit/f7d6cae5e54ca2564288d5df9cd085c907be4ac8